tests/sentry/api/endpoints/test_organization_root_cause_analysis.py

```diff
@@ -243,26 +243,17 @@ def test_returns_change_data_for_regressed_spans(self):
             )
 
         assert response.status_code == 200, response.content
-
-        # Check that sample IDs are gathered, but remove them from the data
-        # for checking since they are randomized
-        assert all("sample_event_id" in row for row in response.data)
-        for row in response.data:
-            del row["sample_event_id"]
```

Comment:
Dropped `sample_event_id` because this was only supposed to be used internally and has no purpose being exposed in the UI.